plugins/woocommerce/changelog/enhance-textdomain-replacement

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Replace woocommerce-blocks textdomain in *.ts, *.tsx and *.json files
```

Comment:
I think the changelog doesn't really correctly reflects the change. Might be good to update before merging.

Reply:
Good catch, @peterfabian. I just updated the changelog entry accordingly.